apis/postgresql/v1alpha1/postgresql_standalone_types.go

```diff
@@ -39,6 +39,8 @@ type PostgresqlStandaloneParameters struct {
 
 // PostgresqlStandaloneSpec defines the desired state of a PostgresqlStandalone.
 type PostgresqlStandaloneSpec struct {
+	ConnectableInstance `json:",inline"`
```

Comment:
Can we have more explicit name for this object? for instance `ConnectionInstanceDetails?`

Reply:
Here I'm just composing with a type that has `WriteConnectionSecretRefTo`. The resulting CRD and API spec will have the property `spec.writeConnectionSecretRefTo` as expected when serializing to json or yaml.